apps/settings/js/panels/improveBrowserOS/panel.js

```diff
@@ -0,0 +1,34 @@
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+
+  return function ctor_improveBrowserOS() {
+    return SettingsPanel({
+      onInit: function(panel) {
+        this.openAppStats = panel.querySelector('#openAppStats');
+        this.openAppStats.addEventListener('click',
+                                          this._onOpenAppStats.bind(this));
+      },
+
+      onUninit: function() {},
+      onShow: function() {},
+      onHide: function() {},
+
+      _onOpenAppStats: function() {
+        var activity = new window.MozActivity({
+          name: 'marketplace-search',
+          data: {
+            type: 'firefox-os-app-stats'
+          }
+        });
+
+        activity.onerror = function() {
```

Comment:
Is this likely to happen? Under what scenarios will it happen? The URL should be "https://marketplace.firefox.com/usage" as per this commit I'm working on: https://github.com/mozilla/fireplace/commit/8ab4e7ee557a#diff-9d9d2e177b183f25634866826d4c75d4R58 - but I'm wondering if we need it. It will open in a browser and create a whole new flow to test. If this isn't needed, I'd rather remove.